metadata.json

```diff
@@ -1,4 +1,6 @@
 {
     "description": "Applications from the LSMO group",
-    "title": "LSMO apps"
+    "title": "LSMO apps",
+    "version": "0.8.6",
```

Comment:
Just replace version with "0.1.0", I've never actually released this package.

Reply:
Done